src/pocketmine/plugin/PluginManager.php

```diff
@@ -248,6 +248,14 @@ public function loadPlugins(string $directory, array $newLoaders = null){
 								continue;
 							}
 
+							if(!empty($redundant = self::findRedundantApis(...$description->getCompatibleApis()))){
+								$messages = [];
+								foreach($redundant as $base => $duplicates){
+									$messages[] = implode(", ", $duplicates) . " (already included with $base)";
+								}
+								$this->server->getLogger()->warning($description->getName() . " declares unnecessary API versions: " . implode(", ", $messages));
```

Comment:
This might be difficult to create translations for.

Reply:
Meanwhile let's have language-specific punctuation. It should be `、` instead of `, ` in Chinese! 😛